WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -230,6 +232,22 @@ private extension TotalsViewModel {
                 connectedReader == nil ? .disconnected: .connected
             }
             .assign(to: &$connectionStatus)
+
+        Publishers.CombineLatest3($connectionStatus.removeDuplicates(),
+                                  $isSyncingOrder.removeDuplicates(),
+                                  $cardPresentPaymentInlineMessage)
```

Comment:
Yes, there are no negative effects. It can be removed. I was trying to solve issues where events from a previous payment would be emitted when quickly going back-and-forth from/to checkout. However, this duplication removal doesn't change anything.

Reply:
Removed in this commit https://github.com/woocommerce/woocommerce-ios/pull/13437/commits/50591cb67b92c2c4b185c5dfb56c7871a7ccf1f6